src/blackmetal/src/communication/controlSW/BlackMetal.hpp

```diff
@@ -39,12 +39,12 @@ class BlackMetal
 	/**
 	 * @brief Retrieve the chassis length set in the config file.
 	 */
-	const double &chassisLength();
+	[[maybe_unused]] const double &chassisLength();
 
 	/**
 	 * @brief Retrieve the wheel radius set from the config file.
 	 */
-	const double &wheelRadius();
+	[[maybe_unused]] const double &wheelRadius();
```

Comment:
Mozno aj takto. Geter by nemal menit stav v classe
```suggestion
	[[maybe_unused]] const double &wheelRadius() const;
```